<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-MB50M7DV" failures="1" tests="4" name="FirefoxTest" time="41.307" errors="0" timestamp="2022-09-17T19:47:32 IST">
  <testcase classname="Practice12.TwitterTestNG" name="A_verifyTwitterCompromisedAccount" time="1.761">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://help.twitter.com/en/safety-and-security/twitter-account-compromise] but found [https://help.twitter.com/en/safety-and-security/twitter-account-compromised]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://help.twitter.com/en/safety-and-security/twitter-account-compromise] but found [https://help.twitter.com/en/safety-and-security/twitter-account-compromised]
at Practice12.TwitterTestNG.A_verifyTwitterCompromisedAccount(TwitterTestNG.java:81)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- A_verifyTwitterCompromisedAccount -->
  <testcase classname="Practice12.TwitterTestNG" name="B_verifyTwitterAccountSecurity" time="1.89"/>
  <testcase classname="Practice12.facebookTestNG" name="verifyGoBackToMessengerButton" time="2.563"/>
  <testcase classname="Practice12.facebookTestNG" name="verifyHelpCenterButton" time="3.63"/>
</testsuite> <!-- FirefoxTest -->
